src/System.Windows.Forms/src/System/Windows/Forms/LinkClickedEventArgs.cs

```diff
@@ -12,11 +12,43 @@ public class LinkClickedEventArgs : EventArgs
         /// <summary>
         ///  Initializes a new instance of the <see cref='LinkClickedEventArgs'/> class.
         /// </summary>
-        public LinkClickedEventArgs(string? linkText)
+        /// <param name="linkText">The text of the link being clicked.</param>
+        public LinkClickedEventArgs(string? linkText) : this(linkText, 0, 0) { }
+
+        /// <summary>
+        ///  Initializes a new instance of the <see cref='LinkClickedEventArgs'/> class.
+        /// </summary>
+        /// <param name="linkText">The text of the link being clicked.</param>
+        /// <param name="linkStart">The start of the link span being clicked.</param>
+        /// <param name="linkLength">The length of the link span being clicked.</param>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// The value for <paramref name="linkStart"/> or <paramref name="linkLength"/> is negative.
+        /// -or-
+        /// The values for <paramref name="linkStart"/> and <paramref name="linkLength"/> would overflow addition.
```

Comment:
To preserve the line breaks we need to wrap it up in `<para>` tags (or have double new line, which isn't obvious)
```suggestion
        /// <para>The value for <paramref name="linkStart"/> or <paramref name="linkLength"/> is negative.</para>
        /// <para>-or-</para>
        /// <para>The values for <paramref name="linkStart"/> and <paramref name="linkLength"/> would overflow addition.</para>
```